streams/src/main/java/org/apache/kafka/streams/processor/internals/TopologyMetadata.java

```diff
@@ -84,14 +84,14 @@
         public AtomicLong topologyVersion = new AtomicLong(0L); // the local topology version
         public ReentrantLock topologyLock = new ReentrantLock();
         public Condition topologyCV = topologyLock.newCondition();
-        public List<TopologyVersionWaiters> activeTopologyWaiters = new LinkedList<>();
+        public List<TopologyVersionListener> activeTopologyUpdateListeners = new LinkedList<>();
```

Comment:
Just renamed from `waiters` to `listeners`

Reply:
Also another quick question regarding why we need to keep `topologyVersion` an atomic long? Seems besides the getters all of its updators are under the lock as well.

Reply:
Good find, yeah I believe it no longer needs to be an AtomicLong, I'll change back to `long`

Reply:
Oh right actually no, we. do still need it to be an AtomicLong as we check it in the StreamThread main loop when looking for topology updates. And obviously we don't want to have to grab the full lock for that